src/fragments/lib/auth/js/advanced.mdx

```diff
@@ -300,50 +373,57 @@ Step 2. Login with `Auth0`, then use the id token returned to get AWS credential
 const { idToken, domain, name, email, phoneNumber } = getFromAuth0(); // get the user credentials and info from auth0
 const { exp } = decodeJWTToken(idToken); // Please decode the id token in order to get the expiration time
 
-Auth.federatedSignIn(
-  domain, // The Auth0 Domain,
-  {
-    token: idToken, // The id token from Auth0
-    // expires_at means the timestamp when the token provided expires,
-    // here you can derive it from the expiresIn parameter provided,
-    // then convert its unit from second to millisecond, and add the current timestamp
-    expires_at: exp * 1000 // the expiration timestamp
-  },
-  { 
-    // the user object, you can put whatever property you get from the Auth0
-    // for example:
-    name, // the user name
-    email, // Optional, the email address
-    phoneNumber, // Optional, the phone number
-  } 
-).then(cred => {
-  console.log(cred);
-});
+async function federatedSignIn () {
```

Comment:
Same as above, rename this to `getCognitoCredentials()`

Reply:
Thanks @nadetastic - change applied